<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>商品管理</string>
     </attribute>
     <widget class="QComboBox" name="cb_selectRow">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="keyWord">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>701</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>筛选</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_selectRel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>关键字：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>701</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_selectRow2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_selectRel2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="keyWord2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_selete2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_search2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>关键字：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>筛选</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
